src/Commands/PluginCommand.php

```diff
@@ -0,0 +1,568 @@
+<?php
+
+/**
+ * The PluginCommand class manages Terminus plugins.
+ */
+
+namespace Pantheon\Terminus\Commands;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Exceptions\TerminusNotFoundException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Manage Terminus plugins.
+ *
+ * @package Pantheon\Terminus\Commands
+ */
+class PluginCommand extends TerminusCommand
+{
+
+    /**
+     * Install one or more Terminus plugins.
+     *
+     * @command plugin:install
+     * @aliases plugin:add
+     *
+     * @option project_list A comma delimited list of one or more plugin projects to install
+     *
+     * @usage <URL to plugin archive, Git or Packagist project 1>,[URL to plugin archive, Git or Packagist project 2],...
+     */
+    public function install($project_list = '')
+    {
+        if (empty($project_list)) {
+            $message = "Usage: terminus plugin:<install|add>";
+            $message .= " <URL to plugin archive, Git or Packagist project 1>,";
+            $message .= "[URL to plugin archive, Git or Packagist project 2],...";
+            $this->log()->error($message);
+            return false;
+        }
+
+        $projects = explode(',', $project_list);
+        $plugins_dir = $this->getPluginDir();
+        foreach ($projects as $project) {
+            $is_url = (filter_var($project, FILTER_VALIDATE_URL) !== false);
+            if (!$is_url) {
+                if (!$this->isValidPackagistProject($project)) {
+                    $message = "{$project} is not a valid Packagist project.";
+                    $this->log()->error($message);
+                } else {
+                    $path = explode('/', $project);
+                    $plugin = $path[1];
+                    if (is_dir($plugins_dir . $plugin)) {
+                        $message = "{$plugin} plugin already installed.";
+                        $this->log()->notice($message);
+                    } else {
+                        exec("composer create-project --dev -n -d {$plugins_dir} {$project}", $output);
```

Comment:
That was a bug in the Terminus Quicksilver Plugin; I simply forgot to push the tags up to the github repository after I tagged 1.1.0. Fixed.